src/components/Offers/Edit/EditOfferSchema.js

```diff
@@ -71,4 +71,5 @@ export default yup.object().shape({
         ),
     isHidden: yup.boolean(HumanValidationReasons.BOOLEAN),
     owner: yup.string(),
+    applyURL: yup.string().nullable(true),
```

Comment:
use `matches` and the same regex used in the backend.

Maybe some of this regexes would also be usefull to put in the commons package @Naapperas

Reply:
OK, on it

Reply:
@Naapperas Please put the whole function to test the regex, not just the expressions